src/blocks/card.scss

```diff
@@ -0,0 +1,110 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  box-sizing: border-box;
+  width: 200px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  font-family: Roboto, Arial, Helvetica, sans-serif;
+
+  --main-text-color: #060b35;
+  --secondary-text-color: #616070;
+  --acdc-color: #00acdc;
+  --white-color: #fff;
+
+  padding: 16px;
+
+  &__image {
+    width: 160px;
+    height: 134px;
+    margin-top: 16px;
+    margin-left: 3px;
+  }
+
+  &__product-name {
+    height: 36px;
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    color: var(--main-text-color);
+
+    margin: 0;
+    margin-top: 40px ;
+  }
+
+  &__product-code {
+    margin: 0;
+    font-weight: 400;
+    font-size: 10px;
+    line-height: 14px;
+    color: var(--secondary-text-color);
+
+    margin-top: 4px;
+  }
+
+  &__review {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    height: 16px;
+
+    margin-top: 16px;
+  }
+
+  &__reviews {
+    margin: 0;
+    font-weight: 400;
+    font-size: 10px;
+    line-height: 14px;
+    color: var(--main-text-color);
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+
+    margin-top: 24px;
+
+  }
+
+  &__price-text {
+    margin: 0;
+    font-weight: 400;
+    font-size: 12px;
+    line-height: 18px;
+    color: var(--secondary-text-color);
+  }
+
+  &__price-sum {
+    margin: 0;
+    font-weight: 700;
+    font-size: 16px;
+    line-height: 18px;
+    color: var(--main-text-color);
+  }
+
+  &__button-buy {
+    box-sizing: border-box;
+    width: 166px;
+    height: 40px;
```

Comment:
it's better not hardcode height for elements with content inside. Just use padding: 11px 0;